Jenkinsfile_nightly

```diff
@@ -56,9 +61,22 @@ withNightlyPipeline(type, product, component) {
 
 
     after('securityScan') {
-        def time = "120"
-        echo "Waiting ${time} seconds for deployment to complete prior starting functional testing"
-        sleep time.toInteger() // seconds
+        def sleepTime = (env.BRANCH_NAME.startsWith("PR")) ? '210s' : '120s'
+        sh "sleep ${sleepTime}"
+        if (env.BRANCH_NAME.startsWith("PR")) {
+            withAksClient('nonprod', product) {
+
+                def dockerImage = new DockerImage(product, component, null, env.BRANCH_NAME, env.GIT_COMMIT)
```

Comment:
PR not needed as nightly ruins against aat

Reply:
These changes are needed if we want to run PR against nightly build. So keeping as it is :)